pkgs/os-specific/linux/wpa_supplicant/default.nix

```diff
@@ -113,7 +113,7 @@ stdenv.mkDerivation rec {
     substituteInPlace Makefile --replace /usr/local $out
     export NIX_CFLAGS_COMPILE="$NIX_CFLAGS_COMPILE \
       -I$(echo "${lib.getDev libnl}"/include/libnl*/) \
-      -I${lib.getDev pcsclite}/include/PCSC/"
+      ${optionalString (pcsclite != null) "-I${lib.getDev pcsclite}/include/PCSC/"}"
```

Comment:
Overwriting random packages to null is an anti pattern and generally does not work. Please add an option named something like withPcsclite instead.

Reply:
Fixed!